manim/mobject/mobject.py

```diff
@@ -78,6 +78,7 @@ class Mobject:
     """
 
     animation_overrides = {}
+    submobjects = []
```

Comment:
What was the reasoning for adding this class level attribute? Also, why did you decide against this in `opengl_mobject.py`?

Reply:
This was an ugly workaround similar to the one pointed out in the previous review. I've removed it.